apis/conversion/src/index.js

```diff
@@ -0,0 +1,63 @@
+import hypercube from './hypercube';
+
+const getType = async ({ halo, name, version }) => {
+  const { types } = halo;
+  const SN = await types
+    .get({
+      name,
+      version,
+    })
+    .supernova();
+  return SN;
+};
+
+const getPath = (qae) => {
+  return qae && qae.data && qae.data.targets && qae.data.targets[0] && qae.data.targets[0].propertyPath;
+};
+
+const getDefaultExportPropertiesFnc = (path) => {
```

Comment:
I would prefer to call it `getDefaultExportPropertiesFn` instead, and that goes for all of the functions here ending with `Fnc`, rename to `Fn` instead...

Reply:
Fixed!